google/cloud/bigquery/client.py

```diff
@@ -4029,7 +4029,7 @@ def list_rows(
 
         # No schema, but no selected_fields. Assume the developer wants all
         # columns, so get the table resource for them rather than failing.
-        elif len(schema) == 0:
+        elif not schema:
```

Comment:
Same here. Let's be careful about breaking changes regarding `table.schema` allowing `None`.

Reply:
I also switched this back to the old check.